js/eventPage.js

```diff
@@ -0,0 +1,20 @@
+// added from "chrome tab count"
+
+function updateIcon(tabId, changeInfo, tab) {
+    chrome.tabs.query({currentWindow: true},
+    // chrome.tabs.query({},
```

Comment:
Please remove unused code (Lin5), empty lines (L7 & L9), and align indent (L6~12).